### PR TITLE
Removed misspellings from docs

### PR DESCRIPTION
**Pull request by deborahvandervegt**

### CODE REVIEW

**Comment by niels-nijens** [comment]
Thanks for contributing! I'll review the changes later today.

File: docs/01-Getting-started.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ $ composer require accompli/accompli:dev-master --dev
 
 ## Using Accompli
 
-In order for Accompli to run you require a accompli.json file in the root of your project. 
+In order for Accompli to run your project requires an accompli.json file in the root of your project.
```
Comment (niels-nijens) [inline docs/01-Getting-started.md]: Might be better to change 'your project' to 'the project' at the end of the sentence.

File: docs/01-Getting-started.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ For more detailed information on how to configure your accompli.json, please see
 See the [tasks](03-Tasks.md) documentation to learn what tasks are provided by Accompli and how to create your own tasks.
 
 ### Install a release for deployment
-To install a release for deployment you need to run the following command from your project root:
+To install a release for deployment you'll need to run the following command from your project root:
```
Comment (niels-nijens) [inline docs/01-Getting-started.md:78]: This doesn't look like an improvement to me.

File: docs/01-Getting-started.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ The following extensions are suggested to speed up SSH related tasks:
 * mcrypt
 * openssl
 
-In order to install releases from version control, you will need to have git or subversion 
+In order to install releases from version control, you'll need to have git or subversion
```
Comment (niels-nijens) [inline docs/01-Getting-started.md:23]: This isn't a typo fix, but rather shortening 'will'.

File: docs/02-Configuration.md [hunk 4]
```diff
@@ -58,7 +58,7 @@ To configure a task as an event subscriber add your class to the 'events.subscri
 }
 ```
 
-If your task has constructor arguments to configure it's behavior, you're able to configure them next to the class name of the task:
+If your task has constructor arguments to configure it's behavior, you'll be able to configure them next to the class name of the task:
```
Comment (niels-nijens) [inline docs/02-Configuration.md:61]: This doesn't look like an improvement to me.

File: docs/03-Tasks.md [hunk 5]
```diff
@@ -67,7 +67,7 @@ class ExampleTask extends AbstractConnectedTask
 
 ### Dispatched Accompli events
 
-The amount of events dispatched depends on the configured deployment strategy. A typical deployment strategy should at least dispatch the events described below.
+The amount of events dispatched depends on the configured deployment strategy. A typical deployment strategy should at least to dispatch the events described below.
```
Comment (niels-nijens) [inline docs/03-Tasks.md]: This is a typo.

File: docs/01-Getting-started.md (hunk 2 above)
Comment (deborahvandervegt) [inline docs/01-Getting-started.md:78]: Small change but this makes it less demanding in my opinion.

File: docs/01-Getting-started.md (hunk 3 above)
Comment (deborahvandervegt) [inline docs/01-Getting-started.md:23]: Contributes to the readability of the doc.

Comment (niels-nijens) [inline docs/01-Getting-started.md:23]: Agreed.

File: docs/01-Getting-started.md (hunk 2 above)
Comment (niels-nijens) [inline docs/01-Getting-started.md:78]: Agreed.

**Pull request merged**